app/models/user.rb

```diff
@@ -138,6 +138,7 @@ def account_price_groups
   # purchasing that product
   def accounts_for_product(product)
     acts = accounts.active.for_facility(product.facility).to_a
+    puts acts.map { |acct| [acct, acct.validate_against_product(product, self)] }
```

Comment:
Seems like this snuck through and it shouldn’t be here, no? 😄

Reply:
Oops.